interop_binaries/src/lib.rs

```diff
@@ -35,6 +35,11 @@ use trillium_api::ApiConnExt;
 use url::Url;
 
 #[cfg(feature = "testcontainer")]
+// Consts in this module are derived from `include_bytes!()`. If the target file doesn't exist at
+// lint-time, they'll be empty, so this lint does us no good. This lint is only on Rust >1.79, so
+// we have to ignore unknown lints for MSRVs.
```

Comment:
nit: In the "skip" case, the file will still exist, but it will be empty. (I think `include_bytes!()` on a nonexistent file would error)